sdk/storage/platform-matrix-all-versions.json

```diff
@@ -3,11 +3,11 @@
     "Agent": {
       "ubuntu-20.04": {
           "OSVmImage": "env:LINUXVMIMAGE",
-          "Pool": "azsdk-pool-mms-ubuntu-2004-storage"
+          "Pool": "env:LINUXPOOL"
```

Comment:
@benbp this is what caused the storage tests-weekly matrix to not generate correctly.